db/version_set.cc

```diff
@@ -122,7 +121,7 @@ class FilePicker {
     }
   }
 
-  int GetCurrentLevel() { return returned_file_level_; }
+  int curr_level() const { return curr_level_; }
```

Comment:
Sure. I'll revert it to avoid confusion. In the next TechDebt Week, maybe we should clear up these functions to follow the convention mentioned in Google C++ Style Guide: https://google.github.io/styleguide/cppguide.html#Function_Names . Now we have both cases in the code base.